TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -24,19 +27,18 @@
     "schema": {
         "username": {
             "type": "string",
-            "required": False,
+            "required": True,
```

Comment:
Interesting - why change this?  I'm guessing it was a bug to be optional before?

Reply:
Yeah, I think this was from when I had a misunderstanding about what was included in the secret before--I originally thought it was username & password.

Reply:
It's much simpler now that it's username and one of [password, password_from_secret_arn]